src/main.py

```diff
@@ -121,7 +121,7 @@ def main():
     parser.add_argument('-O', '--output-device', default='default',
                         help='Name of the audio output device')
     parser.add_argument('-T', '--trigger', default='gpio',
-                        help='Trigger to use {\'clap\', \'gpio\'}')
+                        help='Trigger to use {"clap", "gpio", "ok-google"}')
```

Comment:
We can do: choices=['clap', 'gpio', 'ok-google'],
https://docs.python.org/2.7/library/argparse.html#choices